lessons/10_debugging/classwork.md

```diff
@@ -0,0 +1,149 @@
+# Lesson 10 Classwork
+
+Let's practice debugging!
+
+Find the bugs!
+
+## Spec
+
+Create a simple web page for every exercise and fix the code.
+
+### Print numbers 1-10
+
+Expected Output: `"1 2 3 4 5 6 7 8 9 10"`
+
+```html
+<button onclick="button1()">Print numbers 1-10</button>
+```
+
+```js
+function button1() {
+  var string = ''
+  for (var i = 0; i < 12; i++) {
+    string = string + i + ', '
+  }
+  string = string.substring(0, string.length - 2)
+  console.log(string)
+}
+```
+
+### Count by 10's
+
+Expected Output: `"[0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100]"`
+
+```html
+<button onclick="button2()">Count by 10's</button>
+```
+
+```js
+function button2() {
+  var counter = 0
+  var index = 0
+  var arrayOf10s = []
+  while (counter <= 100) {
+    arrayOf10s[index] = counter
```

Comment:
this changes to use `.push()` right?

Reply:
Yes, they need to fix it and solve the problem via debugging